docs/generated/oc_by_example_content.adoc

```diff
@@ -776,6 +776,28 @@ Create a new secret based on a key file or on files within a directory
 ====
 
 
+== oc secrets new-sourcecfg
+Create a new source secret
```

Comment:
@csrwng the cacert is used to authenticate the server you're talking to is who it says it is.  the ssh key is used to authenticate to the server.  right?